physlr/physlr.py

```diff
@@ -1555,22 +1519,35 @@ def determine_molecules_extensive(g, u):
     @staticmethod
     def determine_molecules(g, u, strategy):
         """Assign the neighbours of this vertex to molecules."""
-        communities = []
-        if strategy == 1:  # bi-connected
-            communities = \
-                Physlr.detect_communities_biconnected_components(g, g[u].keys())
-        elif strategy == 2:  # bi-connected + k-clique
-            communities = Physlr.determine_molecules_bc_k_cliques(g, u)
-        elif strategy == 3:  # bi-connected + Louvain
-            communities = Physlr.determine_molecules_bc_louvain(g, u)
-        elif strategy == 4:  # bi-connected + sqCos
-            communities = Physlr.determine_molecules_bc_cosine_of_squared(g, u)
-        elif strategy == 5:  # bi-connected + partition + bi-connected + k-cliques + merge
-            communities = Physlr.determine_molecules_partition_split_merge(g, u)
-        elif strategy == 6:  # extensive detection
-            communities = Physlr.determine_molecules_extensive(g, u)
-        else:
-            exit("\033[93m Wrong input argument: --separation-strategy!\033[0m")
+        alg_list = strategy.split(".")
```

Comment:
```suggestion
        alg_list = strategy.split("+")
```
Let's use `+` rather than `.`.